manifest.json

```
@@ -6,13 +6,12 @@
         "en": "LibreOffice-based online office suite with collaborative editing",
         "fr": "Suite office en ligne et collaborative, basée sur LibreOffice"
     },
-    "version": "6.4.10~ynh2",
+    "version": "21.11~ynh1",
```

```suggestion
    "version": "21.11.0.6~ynh1",
```
As the upgrade is triggered by changing the app version we may want to use the real version installed

---

Since it's mainly a debian package, I thought that it would be upgraded by standard `apt upgrade` to the system, is it not the case? Is the package marked as hold?